snmp/os-updates.sh

```diff
@@ -32,10 +33,11 @@ if [ -f /etc/os-release ]; then
 		else
 			echo "0";
 		fi
-	elif [ $OS == "ubuntu" ]; then
-		UPDATES=`$BIN_APT $CMD_APT | $BIN_WC $CMD_WC`
+	elif [ $OS == "ubuntu" ] || [ $OS == "debian" ]; then
+		`$BIN_APT $CMD_UPDATE`
```

Comment:
Why the backquotes on this?  If you don't want the output, just >/dev/null - it's clearer.